app/plugin_api_test.go

```diff
@@ -620,6 +619,32 @@ func TestPluginAPIRemoveTeamIcon(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestPluginAPIUpdateUserActive(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	api := th.SetupPluginAPI()
+
+	err := api.UpdateUserActive(th.BasicUser.Id, true)
+	require.Nil(t, err)
+	user, err := api.GetUser(th.BasicUser.Id)
+	require.Nil(t, err)
+	require.NotNil(t, user)
+
+	err = api.UpdateUserActive(th.BasicUser.Id, false)
+	require.Nil(t, err)
+	if _, err = api.GetUser(th.BasicUser.Id); err!=nil{
```

Comment:
Yeah, something's off about this: looking at the code, `GetUser` shouldn't fail just because the user has been deactivated. @akhilanandbv003, suggest a pattern like this:
```go
err = api.UpdateUserActive(th.BasicUser.Id, false)
require.Nil(t, err)
user, err = api.GetUser(th.BasicUser.Id)
require.Nil(t, err)
require.NotEqual(t, 0, user.DeleteAt)
```

and then when re-activating:
```go
err = api.UpdateUserActive(th.BasicUser.Id, true)
require.Nil(t, err)
user, err = api.GetUser(th.BasicUser.Id)
require.Nil(t, err)
require.Equal(t, 0, user.DeleteAt)
```

Reply:
Made changes accordingly as per your suggestion. Also regarding the IDE , I was using Goland and that might have been causing the issue. Switched to VSCode now. Hope it fixes it. Irrespective of which, I have been running `make gofmt` and `make check-style`. Please let me know if you still find some issues with indentation.